test/powershell/Modules/Microsoft.PowerShell.Utility/Read-Host.Tests.ps1

```diff
@@ -38,6 +38,18 @@ Describe "Read-Host Test" -tag "CI" {
         [pscredential]::New("foo",$result).GetNetworkCredential().Password | Should -BeExactly TEST
     }
 
+    It "Read-Host returns a string when using -MaskInput parameter" {
+        $result = $ps.AddScript("Read-Host -MaskInput").Invoke()
+        $result | Should -Be $th.UI.ReadLineData
+    }
+
+    It "Read-Host throws an error when both -AsSecureString parameter and -MaskInput parameter are used" {
+        # Contrary to the rest of the tests this does not need to be invoked through a runspace since it is going to throw an error.
+        $errorId = "AmbiguousParameterSet,Microsoft.PowerShell.Commands.ReadHostCommand"
+        {Read-Host -MaskInput -AsSecureString} | Should -Throw -ErrorId $errorId
+    }
+
+
```

Comment:
Extra line.
```suggestion
```

Reply:
fixed in 198b33a4a8ee1e69fdb23b1a242458cafdb5fb7a